lib/blend.ex

```diff
@@ -74,13 +74,11 @@ defmodule Blend do
   end
 
   def clean_lockfiles do
-    blend_names =
-      blends()
-      |> Map.keys()
+    blend_names = blends() |> Map.keys()
 
-    Path.wildcard("#{@blend_dir}/*.mix.lock")
+    Path.wildcard("#{@blend_dir}/*/mix.lock")
     |> Enum.reject(fn path ->
-      path in Enum.map(blend_names, fn name -> "#{@blend_dir}/#{name}.mix.lock" end)
+      path in Enum.map(blend_names, fn name -> "#{@blend_dir}/#{name}/mix.lock" end)
```

Comment:
This folder restructuring made me think that maybe we could improve `mix blend.clean` to also clean the artifacts. Which means directly removing the blend folder, after this restructure.

Something I can tackle later as a follow up. Just sharing.